backend/API_DOC.md

```diff
@@ -195,6 +195,41 @@ Endpoint to create and errand.
 ---
 
 ## Request
+### Change Request Status
+
+Endpoint to change the status on a request.
+
+**URL** : `http://api.good-grocer.click/request/update-status`
+
+**Method** : `POST`
+
+**Auth Required** : YES
+
+**Body Parameters** :
+
+```json
+{
+  "id": "[int, required id of the item]",
```

Comment:
Maybe change the description to `id of request` to make it more clear. I though it was the item id lmao

Reply:
ohh yes thanks for catching that, forgot to change it when i copy and pasted lmao